weblab_cg/tests/test_chaste_cg_develop.py

```diff
@@ -270,6 +270,7 @@ def _to_equation(self, equation_str):
         """ Do needed substitutions and parse brackets to handle conditionals (cond?exp:exp) """
         # pow->Pow ceil -> ceiling in sympy
         equation_str = equation_str.replace("pow(", 'Pow(').replace("ceil(", 'ceiling(').replace('fabs(', 'Abs(')
+        equation_str = equation_str.replace('M_PI', '3.141592')
```

Comment:
This seems a very low precision version of pi!

Reply:
actually pi will probably do it